### PR TITLE
added color coding, verbose mode, made refinements across all 3 files

### PR DESCRIPTION
**Pull request by m0mosenpai**
# Description

Added color coding, connected all 3 files for a finished MVP, made small formatting and other adjustments.

## Related Issues

Extensions of #1, #2 and #4 Issues 

## Type of Change

- [x] Enhancements (non-breaking change which adds functionality)

# Checklist

- [x] The pull request is for branch `develop`
- [x] `black` gave 0 errors
- [x] Squashed commits into a single one

# Screenshots of working

image
Color coding and verbose mode sample.

### CODE REVIEW

File: clipbit.py [hunk 1]
```diff
@@ -23,13 +27,34 @@
...
+            if detailed:
+                tabulate_summary(final_summary, captions)
+            else:
+                print(f"\n{final_summary}\n")
+            verboseprint(
+                "\033[96m\u2192 Captions, Script and Summary are all available under videos/<video_name>/ directory.\033[0m"
+            )
+
+
+def tabulate_summary(summary, captions):
```
Comment (KimaruThagna) [inline clipbit.py]: What does this function do? Remember to insert docstrings. This allows the reviewer to have a good understanding of what your function attempts to achieve

**Review by KimaruThagna — APPROVED** [review]
In terms of code and logic I have no issue. LGTM
I suggest however that you implement the docstrings.

**Comment by m0mosenpai** [comment]
image

New preview after changes

**Review by KimaruThagna — APPROVED** [review]
LGTM

**Review by Miguel-Enrique13 — APPROVED** [review]

**Pull request merged**